### ISSUE TITLE
Create rule S6823 (`jsx-a11y/aria-activedescendant-has-tabindex`): DOM elements with the `aria-activedescendant` property should be accessible via the tab key

### ISSUE THREAD

**Issue by ilia-kebets-sonarsource** [SonarSource/SonarJS#4235]
[plugin rule](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/aria-activedescendant-has-tabindex.md#jsx-a11yaria-activedescendant-has-tabindex)

https://github.com/SonarSource/rspec/pull/3275
